cpp/src/barretenberg/honk/flavor/ultra.hpp

```diff
@@ -35,12 +36,15 @@ class Ultra {
     using GroupElement = G1::element;
     using Commitment = G1::affine_element;
     using CommitmentHandle = G1::affine_element;
-    using PCSParams = pcs::kzg::Params;
+    // Note: UltraHonk will be run with KZG by default but temporarily we set the commitment to IPA to
+    // be able to do e2e tests with this pcs as well
+    using PCSParams = pcs::ipa::Params;
+    using PCS = pcs::ipa::IPA<PCSParams>;
 
     static constexpr size_t NUM_WIRES = CircuitConstructor::NUM_WIRES;
     // The number of multivariate polynomials on which a sumcheck prover sumcheck operates (including shifts). We often
     // need containers of this size to hold related data, so we choose a name more agnostic than `NUM_POLYNOMIALS`.
-    // Note: this number does not include the individual sorted list polynomials.
+    // Note: this number does not include the individual ultra list polynomials.
```

Comment:
What did you mean? We have sorted list polynomials as part of plookup argument

Reply:
Wups, accidental renaming, thanks for spotting!